federation/pkg/federation-controller/util/cluster_util.go

```diff
@@ -146,3 +149,55 @@ func GetClientsetForCluster(cluster *federation_v1beta1.Cluster) (*fedclientset.
 	}
 	return nil, err
 }
+
+// Returns a bool to indicate if the object should be forwarded to a cluster
+func SendToCluster(cluster *federation_v1beta1.Cluster, objMeta metav1.ObjectMeta) bool {
```

Comment:
I updated the util functions to take the lower level parameters (Labels  & map[string]string) to simplify the tests, etc.